HW_1/task4.py

```diff
@@ -0,0 +1,6 @@
+# Петя, Катя и Сережа делают из бумаги журавликов. Вместе они сделали S журавликов. Сколько журавликов 
+# сделал каждый ребенок, если известно, что Петя и Сережа сделали одинаковое количество журавликов, а 
+# Катя сделала в два раза больше журавликов, чем Петя и Сережа вместе?
+
+n = int(input("Введите количество журавликов: "))
+print(f'Петя {n//6} Катя {n//6*4} Сережа {n//6}')
```

Comment:
```suggestion
print(f'Петя {n // 6} Катя {n // 6 * 4} Сережа {n // 6}')
```
У вас трижды повторяется одно и то же действие. Для этого придумали переменные.